projects/ngx-charts-on-fhir/src/lib/data-layer/data-layer-color.service.ts

```diff
@@ -22,22 +24,16 @@ export class DataLayerColorService {
     }
   }
 
-  setAnnotationColor(annotation: any, color?: string): void {
-    const line = annotation;
-    line.backgroundColor = color + '33'; // temporary dirty hack to set opacity. assumes color is in 6-character hex format.
-  }
-
   addTransparency(color: string | undefined): string | undefined {
-    if (typeof color === 'string' && !color.endsWith('33')) {
-      return color + '33';
-    }
-    return color;
+    const newcolor = tinycolor(color);  
+    newcolor.setAlpha(.5);
+    return newcolor.toString();
```

Comment:
`.5` is much darker than hex `33`. it's hard to see the lines when the annotation is so dark.

```suggestion
    const newcolor = tinycolor(color);  
    newcolor.setAlpha(.2);
    return newcolor.toString();
```

Reply:
Yes, I will change it as you suggested.